TGC/decisions/Italy-Unification.txt

```diff
@@ -1728,6 +1728,7 @@ political_decisions = {
 					days = 30
 				}
 			}
+			#TODO: Add event for Two Sicilies and the Papal States to propose unification against the Austrian threat
```

Comment:
I still don't think this would have been particularly likely, though certainly they should get events notifying them and allowing them to turn to France for aid for example.